src/navigators/ConnectSignUpStackNavigator.js

```diff
@@ -11,5 +18,12 @@ const ConnectSignUpStackNavigator = StackNavigator({
   ConnectSignUp: {
     screen: ConnectSignUpScreen,
   },
+  ConnectProfile: {
+    screen: ProfileScreen,
```

Comment:
why does the profile screen need to be in the connect signup stack nav?

Reply:
I think it has to be there since the other `ProfileScreen` would be part of the TabNavigator which will interact differently

Reply:
Connect needs to use the `ProfileScreen` in order to display veteran and PO profiles